snmp/tests/test_profiles.py

```diff
@@ -129,6 +129,15 @@ def test_cisco_voice(aggregator):
     aggregator.assert_metric('snmp.{}'.format("cccaPimStatus"), metric_type=aggregator.GAUGE, tags=pim_tags)
     aggregator.assert_metric('snmp.{}'.format("sysUpTimeInstance"), metric_type=aggregator.GAUGE, tags=tags, count=1)
 
+    routers = [
+        "cccaRouterAgentsLoggedOn",
+        "cccaRouterCallsInProgress",
+        "cccaRouterCallsInQueue",
+    ]
+
```

Comment:
Can we add these metrics to the `metrics.py` test file? Since these metrics are in the base cisco profile and can be used in the future

Reply:
I added these metrics to `metrics.py` in a base cisco voice section